galaxy_ng/app/models/auth.py

```diff
@@ -18,8 +22,7 @@
 
 class User(auth_models.AbstractUser):
     """Custom user model."""
-
-    pass
+    auth_provider = models.CharField(_('auth provider'), max_length=60, blank=True)
```

Comment:
where does this field gets set? does this data come from the UI?

Reply:
@rochacbruno I was working with @ShaiahWren yesterday, I walked through the model edit + makemigrations, but then I realized this field is never populated in the database, just computed. 

@ShaiahWren I think if `auth_provider` is removed from UserSerializer.Meta and then spin down/up the containers, it should run without errors. If that all works then you can remove the model edit and the 0026 migration.

Reply:
@awcrosby auth_provider is removed from UserSerializer.Meta and I get this error:
![Screen Shot 2021-10-04 at 12 13 16 PM](https://user-images.githubusercontent.com/64337863/135887092-89dc843c-1ff6-4904-9ce2-bcb7ae82f53a.png)